### DISCUSSION TITLE
woocommerce membership plugin

### DISCUSSION THREAD

**Discussion by MLIKI-Project** [inspireui/support#5586] [New Feature/Ideas Requests]
please add woocommerce membership plugin

**Comment by bakpaorabbit** [comment]
fluxstore already support some membership plugins. I forgot what plugin, but you can check the documentation.

**Reply by MLIKI-Project** [reply]
i used woocommerce membership

**Reply by naturaw-canine-food** [reply]
and did you get it working?

**Reply by naturaw-canine-food** [reply]
> fluxstore already support some membership plugins. I forgot what plugin, but you can check the documentation.

yeah but theyre crap
